force-app/main/default/classes/ADDR_Addresses_UTIL_TEST.cls

```diff
@@ -0,0 +1,1341 @@
+        Test.stopTest();
+
+        System.assertEquals(true, stringEquals);
+    }
+
+    @isTest
+    static void testEqualsCaseSensitiveOnlyStr1NullFalse() {
+        Test.startTest();
+        Boolean stringEquals = ADDR_Addresses_UTIL.equalsCaseSensitive(null, 'test string');
+        Test.stopTest();
+
+        System.assertEquals(false, stringEquals);
+    }
+
+    @isTest
+    static void testEqualsCaseSensitiveOnlyStr2NullFalse() {
+        Test.startTest();
+        Boolean stringEquals = ADDR_Addresses_UTIL.equalsCaseSensitive('test string', null);
+        Test.stopTest();
+
+        System.assertEquals(false, stringEquals);
+    }
+
+    @isTest
+    static void testEqualsCaseSensitiveNotNullEqualsTrue() {
+        Test.startTest();
+        Boolean stringEquals = ADDR_Addresses_UTIL.equalsCaseSensitive('test string', 'test string');
+        Test.stopTest();
+
+        System.assertEquals(true, stringEquals);
+    }
+
+    @isTest
+    static void testEqualsCaseSensitiveNotNullEqualsFalse() {
+        Test.startTest();
+        Boolean stringEquals = ADDR_Addresses_UTIL.equalsCaseSensitive('test string', '!test string');
+        Test.stopTest();
+
+        System.assertEquals(false, stringEquals);
+    }
+
+    /**************************************************************************************************************************
+     ****************************************************** isAddressMatchFuzzy ***********************************************
+     **************************************************************************************************************************/
+    //TODO: The method is not test visible.
+    /*@isTest
+    static void testisAddressMatchFuzzyAddrNewNullFalse() {
+        Test.startTest();
+        Boolean isAddressMatchFuzzy = ADDR_Addresses_UTIL.isAddressMatchFuzzy(null, null, null);
+        Test.stopTest();
+
+        System.assertEquals(false, isAddressMatchFuzzy);
+    }*/
+
+    /**************************************************************************************************************************
+     ****************************************************** strCleanup ********************************************************
+     **************************************************************************************************************************/
+    //TODO: The method is not test visible.
+
+    /**************************************************************************************************************************
+     ****************************************************** copyAddressObj ****************************************************
+     **************************************************************************************************************************/
+    @isTest
+    static void testCopyAddressObjBothNull() {
+        //To Refactor: we don't have guard clause for this.
```

Comment:
Regardless, capture the expected behavior and assert for it, making a note on the method.